src/cmd/devdraw/mkwsysrules.sh

```diff
@@ -49,7 +49,7 @@ echo 'X11='$X11
 echo 'X11H='$X11H
 
 if [ $WSYSTYPE = x11 ]; then
-	echo 'CFLAGS=$CFLAGS '$X11H
+	echo 'CFLAGS=$CFLAGS '$X11H -lXfixes
```

Comment:
Same here, this seems to work correctly:

```suggestion
	echo 'CFLAGS=$CFLAGS '$X11H
	echo 'LDFLAGS=$LDFLAGS -lXfixes'
```